src/ios/Fingerprint.swift

```diff
@@ -23,13 +23,24 @@ import LocalAuthentication
     var reason = "Authentication";
     let data  = command.arguments[0] as AnyObject?;
 
-    if let clientId = data?["clientId"] as! String? {
-      reason = clientId;
-    }
+    var policy:LAPolicy = .deviceOwnerAuthenticationWithBiometrics;
+    if let disableBackup = data?["disableBackup"] as! Bool? {
+        if disableBackup {
+            authenticationContext.localizedFallbackTitle = "";
+            policy = .deviceOwnerAuthenticationWithBiometrics;
+        }else{
+          if let localizedFallbackTitle = data?["localizedFallbackTitle"] as! String? {
+            authenticationContext.localizedFallbackTitle = localizedFallbackTitle;
+          }
 
+          if #available(iOS 9.0, *) {
+              policy = .deviceOwnerAuthentication;
```

Comment:
The issue with this piece of code is that in ios 9+ devices if you don't set the authenticationContext.localizedFallbackTitle as '', iOS will show 'Enter Passcode' button on non-matching finger print, but on clicking there will be no action. The purpose of the initial commit was to fix this issue. Also authenticationContext.localizedFallbackTitle set to '' is enough for disable backup in ios9+ devices. This basically removed the 'Enter Passcode' button from the native popup. So I also suggest that disable backup and policy should be set separately, how this was before